backend/src/main/java/com/trainme/app/user/entity/User.java

```diff
@@ -0,0 +1,24 @@
+package com.trainme.app.user.entity;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@RequiredArgsConstructor
+@Getter
+@Entity
+public class User {
```

Comment:
There should be an empty constructor for an `@Entity` class.

Reply:
Is it so, even if we have @RequiredArgsConstructor annotation?